### PR TITLE
Removing sticky from table checkboxes to prevent chrome crash

### PR DESCRIPTION
**Pull request by LFDanLu**
https://bugs.chromium.org/p/chromium/issues/detail?id=1359961

Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-stately/layout/src/TableLayout.ts [hunk 1]
```diff
@@ -266,7 +268,7 @@ export class TableLayout<T> extends ListLayout<T> {
     let {height, isEstimated} = this.getEstimatedHeight(node, width, this.rowHeight, this.estimatedRowHeight);
     let rect = new Rect(x, y, width, height);
     let layoutInfo = new LayoutInfo(node.type, node.key, rect);
-    layoutInfo.isSticky = node.props?.isSelectionCell;
+    layoutInfo.isSticky = !this.disableSticky && node.props?.isSelectionCell;
```
Comment (LFDanLu) [inline packages/@react-stately/layout/src/TableLayout.ts:271]: Note that the column header checkbox is still sticky, didn't seem to crash when testing so I opted to keep it in

**Comment by LFDanLu** [comment]
NOTE: do NOT merge main in, this PR is only for testing but the patch will be done from the branch

File: packages/@react-stately/layout/src/TableLayout.ts [hunk 2]
```diff
@@ -439,4 +441,22 @@ export class TableLayout<T> extends ListLayout<T> {
 
     return res;
   }
+
+  // Checks if Chrome version is 105 or greater
+  private checkChrome105() {
```
Comment (ktabors) [inline packages/@react-stately/layout/src/TableLayout.ts:446]: This feels like it should belong in util, not that anything else will use this.

Comment (LFDanLu) [reply packages/@react-stately/layout/src/TableLayout.ts:446]: It's pretty specific and temporary so I kept it in here. Also this means we don't have to pull in another dep if this were in aria/utils

**Review by ktabors — APPROVED** [review]

**Pull request merged**